### DISCUSSION TITLE
Few-Shot Prompting 支持情况

### DISCUSSION THREAD

**Discussion by yanghanlin** [lobehub/lobehub#1431] [General Question] [answered]
请问 Lobe Chat 目前在 agent 设定中是否支持类似 NextChat (ChatGPT-Next-Web) 中[预设提示词](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/blob/99fb9dcf11f91fc5ca4eff5a24adb53bfc242fa0/docs/user-manual-cn.md#%E4%BB%80%E4%B9%88%E6%98%AF%E9%9D%A2%E5%85%B7%E5%AE%83%E5%92%8C%E6%8F%90%E7%A4%BA%E8%AF%8D%E7%9A%84%E5%8C%BA%E5%88%AB%E6%98%AF%E4%BB%80%E4%B9%88)的功能，用于在每段对话开头自动添加一些示例消息用于引导模型（[Few-Shot Prompting](https://www.promptingguide.ai/techniques/fewshot)）？

目前似乎 UI 和文档中都没有相关的内容，只能把预设提示词加入角色设定；搜索到代码中定义的 agent 设定中有 `fewShots` 字段

https://github.com/lobehub/lobe-chat/blob/7d3c1b6ea6c3561022e39226d906ee3618e966d2/src/types/agent/index.ts#L30

但是没有搜索到相关 issue / PR，不太了解该功能的开发状态。

**Comment by arvinxx** [comment, marked as answer]
最早是想做这个的，然后加了类型定义。但目前还没开发，然后现在想清楚了会往这个方向来做： https://github.com/lobehub/lobe-chat/discussions/869 。
